src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html

```diff
@@ -57,16 +57,16 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
                 <div class="form-group-narrow" *ngIf="!isExamMode">
                     <jhi-date-time-picker
                         name="assessmentDueDate"
-                        [error]="fileUploadExercise.assessmentDueDateError"
+                        [error]="fileUploadExercise.assessmentDueDateError!"
                         labelName="{{ 'artemisApp.exercise.assessmentDueDate' | artemisTranslate }}"
                         (valueChange)="validateDate()"
-                        [(ngModel)]="fileUploadExercise.assessmentDueDate"
+                        [(ngModel)]="fileUploadExercise.assessmentDueDate!"
                     ></jhi-date-time-picker>
                     <span *ngIf="fileUploadExercise.assessmentDueDateError" class="invalid-feedback">{{ 'artemisApp.exercise.assessmentDueDateError' | artemisTranslate }}</span>
                 </div>
                 <div class="form-group">
                     <label class="form-control-label">{{ 'artemisApp.exercise.includedInOverallScoreLabel' | artemisTranslate }}</label>
-                    <div>
+                    <div *ngIf="fileUploadExercise.includedInOverallScore">
```

Comment:
Isn't this changing the behavior since now you do not show the component at all?

Reply:
I changed it back so it does not change the behaviour